Xamarin.Forms.Core/BindableProperty.cs

```diff
@@ -317,7 +318,7 @@ internal bool TryConvert(ref object value)
 			TypeConverter typeConverterTo;
 			if (SimpleConvertTypes.TryGetValue(valueType, out convertableTo) && Array.IndexOf(convertableTo, type) != -1)
 			{
-				value = Convert.ChangeType(value, type);
+				value = Convert.ChangeType(value, type, System.Globalization.CultureInfo.CurrentCulture);
```

Comment:
With any luck the feature "Use UI Thread Culture When Preforming Conversions" would only require changes in this function. Who knows, maybe this is the only spot! However, right below, we have a `ConvertFromInvariantString` so we'd have to go and understand if that also needs to be updated to take the UI culture.